scripts/populate_users_fixtures.js

```diff
@@ -2,14 +2,52 @@
 const CONFIG = require('config')
 const __ = CONFIG.universalPath
 const _ = __.require('builders', 'utils')
-const { createUserWithItems } = require('../api_tests/fixtures/populate')
-
-createUserWithItems()
-.then(userCreated => {
-  _.success('#### New User available ####')
-  console.log(`Your can now login with :
-  - Username : ${userCreated.username}
-  - Password : 12345678`)
-  return process.exit(0)
-})
-.catch(_.Error('users fixture err'))
+const { createUserWithItems } = require('../tests/api/fixtures/populate')
+const { makeUserAdmin } = __.require('controllers', 'user/lib/user')
+const { makeFriends } = require('../tests/api/utils/relations')
+const { createGroup, addMember, addAdmin } = require('../tests/api/fixtures/groups')
+const user_ = __.require('controllers', 'user/lib/user')
+
+const [ username ] = process.argv.slice(2)
+
+const run = async () => {
+  try {
+    const { _id } = await user_.findOneByUsername(username)
+    console.log(`Username ${username} already exists at ${_id}`)
+    return process.exit(0)
+  } catch (err) {
+    createUserAndGroupAndGFriends(username)
+  }
+}
+
+const createUserAndGroupAndGFriends = async username => {
+  const user = await createUserWithItems({ username })
+  _.info(`${user.username} is a new user`)
+  makeUserAdmin(user._id)
```

Comment:
```suggestion
  await makeUserAdmin(user._id)
```
recommendation level: recommended but not blocking